Pull sprint 10

hiremathpriya opened the pull request:
hey Annika can you check the home-work file and see what I have done so far without using any framework so far. This is our pull request practice as well. Thanks

Commit:
new practice with annika

Commit:
changes

Commit:
some changes

Commit:
for the branch

Commit:
messed up code

Commit:
deleted all the messed up code

Commit:
home-work html

HrishiHiremath commented on home-work/styles.css:
```diff
@@ -0,0 +1,51 @@
+
+.first-block{
+    width: 70%;
```
Hi Priya, since you have an `inline-block` class, you don't need this `display` property here. You could simple use the `inline-block` class on those elements

HrishiHiremath commented on home-work/index.html:
```diff
@@ -0,0 +1,63 @@
+            <div><h1>This is the HTML without any framework</h1></div>
+        </div>
+    
+        <div class="first-block">
```
Hi Priya, if you removed the `display: inline-block` from the class `first-block` then you can add two classes above like `class="first-block inline-block"`

HrishiHiremath reviewed (changes requested):
Hi Priya,

Can you please complete couple of those changes I have requested?

Thanks
Hrishi

Commit:
changes requested from hrishi,done

Commit:
Some small changes

annika-philipp committed:
Update index.html

Hi Priya, looks great! I've only edited (and already committed I think my changes to your file I think) style-wise as in aligning divs etc. I also found a stray </div> that wasn't needed.
